bio_files_processor.py

```diff
@@ -0,0 +1,142 @@
+import os
+
+
+def convert_multiline_fasta_to_oneline(input_fasta: str, output_fasta: str = None):
+    """
+    Reads the input fasta-file, in which the sequence can be split into several lines,
+    then saves it into a new fasta-file in which each sequence fits into one line
+
+    :param input_fasta: Function convert_multiline_fasta_to_oneline
+    :param output_fasta: new fasta-file in which each sequence fits into one line
+    """
+    if not os.path.isfile(input_fasta):
+        raise ValueError('No such fasta file in directory')
+    if output_fasta is None:
+        output_fasta = 'converted_' + input_fasta
+    else:
+        output_fasta = output_fasta + '.fasta'
+    with open(input_fasta, mode='r') as file, open(output_fasta, mode='w') as out_file:
+        out_file.write(file.readline())
+        for line in file:
+            if '>' in line:
+                out_file.write('\n' + line)
+            else:
+                out_file.write(line.strip())
+
+
+def bring_indexes(all_genes: list[str], matching_genes: list[str], out_neighboor_genes, n_before: int = 1,
+                  n_after: int = 1) -> list[str]:
+    """
+    The function receives as input a list of all genes found in the file,
+    a list of target genes - genes of interest, and then selects genes neighboring the genes of interest,
+    located at a given distance before or after the gene of interest
+
+    :param all_genes: list of all genes that was found in .gbk file
+    :param matching_genes: list of genes - matches genes of interest
+    :param out_neighboor_genes: list to write output - genes thar are neighbours to genes of interest
+    :param n_before: number of genes before (>0). Default value is 1
+    :param n_after: number of genes after (>0). Default value is 1
+    :return: out_neighboor_genes - list to write output - genes thar are neighbours to genes of interest
+    """
+    for gene in matching_genes:
+        gene_index = all_genes.index(gene)
+        for index in range(gene_index - n_before if n_before <= gene_index else 0,
+                           gene_index + n_after + 1 if n_after + gene_index <= (len(all_genes) - 1) else len(
+                               all_genes)):
+            if (all_genes[index] != gene and
+                        all_genes[index] not in out_neighboor_genes and
+                        all_genes[index] not in matching_genes:
```

Comment:
Не закрыта скобка, из-за этого падает с ошибкой:)
```suggestion
                        all_genes[index] not in matching_genes):
```